### PR TITLE
Allow updates with only remove/clear updates and operations to single…

### PR DESCRIPTION
**Pull request by baldersheim**
… value numeric fields to pass

even when feed rejection is in effect.
The rationale is that these updates will not affect memory or disk footprint negatively,
but it can have a significant positive effect.
Note that this will only be allowed for buckets that are in sync.

@geirst and @toregge and @vekterli PR

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/server/feed_reject_helper.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    class DocumentUpdate;
+    class ValueUpdate;
+}
+
+namespace proton {
+
+class FeedOperation;
+class UpdateOperation;
+
+class FeedRejectHelper {
```
Comment (geirst) [inline searchcore/src/vespa/searchcore/proton/server/feed_reject_helper.h:16]: Please add class description.

**Review by geirst — APPROVED** [review]
👍  @toregge please have a look as well.

**Review by toregge — APPROVED** [review]
Note that the operations now allowed will use space in the transaction log. If flushing does not keep up with feeding (i.e. unable to prune transaction log, compact doc store or perform disk index fusion) then we might still run out of disk space.

**Review by vekterli — APPROVED** [review]
:+1:

File: searchcore/src/vespa/searchcore/proton/server/feed_reject_helper.h (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/server/feed_reject_helper.h:20]: Done

**Comment by baldersheim** [comment]
Since these operations allowed are small that should not be an issue.

**Pull request merged**